lib/sageone_api_signer.rb

```diff
@@ -76,4 +77,13 @@ def signing_key
       percent_encode(access_token)
     ].join('&')
   end
+
+  def valid_json?(json)
+    begin
+      JSON.parse(json)
+      return true
+    rescue Exception => e
```

Comment:
http://stackoverflow.com/questions/10048173/why-is-it-bad-style-to-rescue-exception-e-in-ruby